### PR TITLE
FA-17 Create templating for thank you page.

### PR DESCRIPTION
**Pull request by username-david**

### CODE REVIEW

File: magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="parent">
+
+[#assign pollSelector = state.getSelector()!]
+[#if pollSelector?has_content]
+  [#assign targetPoll = cmsfn.contentByPath("/"+pollSelector, "polls")!]
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl]: mind the spacing

File: magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    [#if image?has_content && image.link?has_content]
+      <img src="${image.link}" style="width: 200px; height: 200px;">
+    [/#if]
+</p>
+    [#if targetPoll.name?has_content]
+      <h1>${targetPoll.name}</h1>
+      <p>${targetPoll.question}</p>
+</div>
+
+      <p class="thanks">Thank you so much for your vote!</p>
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl]: I18n supports

File: magnolia-voting-app/src/main/resources/voting-contentApp/templates/pages/thanksTemplate.ftl [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+    [@cms.page /]
+    ${resfn.css("/webresources.*css")}
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/templates/pages/thanksTemplate.ftl]: Should we include all the css files?

Comment (username-david) [reply magnolia-voting-app/src/main/resources/voting-contentApp/templates/pages/thanksTemplate.ftl]: I will include the specific CSS file in the next commit

File: magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl (hunk 2 above)
Comment (username-david) [reply magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollData/poll.ftl]: I have used i18n in the lastest commit

File: magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollsubmitted/pollSubmitted.ftl [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+        [#if targetPoll.name?has_content]
+            <h1>${targetPoll.name}</h1>
+            <p class="question">${targetPoll.question}</p>
+    </div>
+            <p class="thanks">${i18n['components.pollsubmitted.thanks']}</p>
+
+        [#--      For handling the not completely deleted poll issue.--]
+        [#else]
+            The poll has been deleted.
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/pollsubmitted/pollSubmitted.ftl]: labels, messages, notifications... should use i18n

File: magnolia-voting-app/src/main/resources/voting-contentApp/webresources/css/submitPage.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.parent{
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/webresources/css/submitPage.css]: Mind the spacing

File: magnolia-voting-app/src/main/resources/voting-contentApp/webresources/css/submitPage.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    width: 200px;
+    height: 200px;
+    padding: 30px;
+}
+h1 {
+    color: #4EC32F;
+    padding-top: 50px;
+    font-size: 35px;
+}
+.thanks {
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/resources/voting-contentApp/webresources/css/submitPage.css]: try to have meaningful name, what is "thanks" here? thanks, message container? or thanks-content?

**Review by ThuyTo — APPROVED** [review]

**Pull request merged**